collections/hub/other_tests/bulk_transfers/bulk-invalid-timestamp.json

```diff
@@ -338,69 +343,122 @@
                 "exec": [
-                  "expect(callback.headers['fspiop-uri']).to.equal(`/testingtoolkitdfsp/bulkTransfers/${environment.bulkTransferId}`)"
+                  "expect(environment.bulkTransferCallback.headers['fspiop-uri']).to.equal(`/testingtoolkitdfsp/bulkTransfers/${environment.bulkTransferId}`)"
                 ]
               },
               {
                 "id": 6,
                 "description": "payerfsp callback - fspiop-http-method should be PUT",
                 "exec": [
-                  "expect(callback.headers['fspiop-http-method']).to.equal('PUT')"
+                  "expect(environment.bulkTransferCallback.headers['fspiop-http-method']).to.equal('PUT')"
                 ]
               },
               {
                 "id": 8,
                 "description": "payerfsp callback - errorInformation should be set correctly",
                 "exec": [
-                  "expect(callback.body).to.have.property(\"bulkTransferState\")",
-                  "expect(callback.body.bulkTransferState).to.equal(\"REJECTED\")"
+                  "expect(environment.bulkTransferCallback.body).to.have.property(\"bulkTransferState\")",
+                  "expect(environment.bulkTransferCallback.body.bulkTransferState).to.equal(\"REJECTED\")"
                 ]
               }
             ]
           },
           "ignoreCallbacks": false,
           "scriptingEngine": "javascript",
-          "delay": "1000",
           "scripts": {
             "postRequest": {
               "exec": [
-                ""
+                "environment.bulkTransferResponse = null",
+                "environment.bulkTransferCallback = null",
+                "const expectedBulkTransferState = 'REJECTED'",
+                "const callbackPath = `/bulkTransfers/{$environment.bulkTransferId}`",
+                "const callbackName = 'put-bulk-transfer'",
+                "",
+                "// lets get retry config",
+                "const maxRetryCount = Number.parseFloat(environment.RETRY_MAX_ATTEMPTS) || 20",
+                "const maxRetryWait = Number.parseFloat(environment.RETRY_MAX_WAIT_IN_MS) || 250",
+                "",
+                "async function main() {",
+                "  let _response = response",
+                "  let _callback = callback",
+                "",
+                "  for (let retryCount = 0 ; retryCount < maxRetryCount; retryCount++) {",
+                "    console.log(`retry-count=${retryCount+1}, max-count=${maxRetryCount}, max-wait=${maxRetryWait}`)",
+                "    ",
+                "    const isOk = _callback?.body?.bulkTransferState === expectedBulkTransferState;",
+                "    ",
+                "    if (isOk) {",
+                "      console.log(`Everything is OK, let's continue`)",
+                "      environment.bulkTransferResponse = _response",
+                "      environment.bulkTransferCallback = _callback",
```

Comment:
If you don't need to access these variables in other requests, you can use requestVariables.